src/App.jsx

```diff
@@ -14,6 +15,10 @@ const App = () => {
       birthDate: "",
       isFormValid: false,
```

Comment:
I think you missed changing this to `isValid`.

Reply:
done